src/hypothesis/strategies.py

```diff
@@ -928,23 +937,41 @@ def do_draw(self, data):
 
 @cacheable
 @defines_strategy
-def builds(target, *args, **kwargs):
+def builds(*target_and_args, **kwargs):
```

Comment:
It now occurs to me that we should probably name this argument `*callable_and_args`, to banish the word "target" entirely.  Sorry 😞

Reply:
Oh yeah, makes sense.